templates/sag-db-mysql/README.md

```diff
@@ -59,27 +84,26 @@ When importing the composite template in Command Central, you have to attach the
 sagcc exec templates composite import -i template.zip
 ```
 
-To install Database Component Configurator 10.5 on the Command Central node with alias `local`, create a database named `webm` and a database user named `webm` with password `webm`, and create all database components schemas with user `root` and password `root`:
+To install Database Component Configurator 10.5 on the Command Central node with alias `local`, create a database named `webm` and a database user named `webm` with password `webm`, and create database schemas for all products with user `root` and password `root`:
 
 ```bash
 sagcc exec templates composite apply sag-db-mysql \
   db.version=latest repo.product=products-10.5 repo.fix=fixes-10.5 nodes=local \
   db.host=mysql db.admin.username=root db.admin.password=root \
-  db.STR.components=[STR] \
```

Comment:
by default in the template the db.STR.components is empty, so either we nned to change the template or we need to keep this in the example

Reply:
Updated - I added the line back in the readme file.